### ISSUE TITLE
update_profile_background_image: endpoint does not exist

### ISSUE THREAD

**Issue by baeuchle** [tweepy/tweepy#1630]
Trying to use tweepy.update_profile_background_image results in the following error message from twitter:
[{'message': 'Sorry, that page does not exist', 'code': 34}]

Ubuntu 21.04, python3-tweepy 3.6.0-2, python 3.8.10

**Comment by Harmon758** [comment]
`API.update_profile_background_image` was removed with https://github.com/tweepy/tweepy/commit/8dba191518366fb756440302eff86d5a868e0306, released with v3.10.

Please make sure you're using the latest version when reporting issues.

**Comment by baeuchle** [comment]
Well it *is* the currently available version on the current version of Ubuntu, I made sure of that. Still, thanks for the reference.

**Comment by Harmon758** [comment]
You're using Tweepy v3.6.0, which is more than three years old now.
There have been four releases since then, and [Tweepy v3.10.0 should be available for Python 3 on any platform](https://pypi.org/project/tweepy/3.10.0/#files).

**Comment by baeuchle** [comment]
Let me rephrase my statement: it is the version currently included in the current version of Ubuntu. So, let's blame the official Ubuntu repository.

But the funny thing is: update_profile_background_image wasn't actually the function I was looking for. Instead, I needed update_profile_banner which I found not in the API reference, but in the diff you linked above, it being immediately underneath the deleted function. So you actually did solve my problem, even in a roundabout way. Thanks. Is there any sense in adding a ticket for missing documentation?

**Comment by Harmon758** [comment]
It's been added with https://github.com/tweepy/tweepy/commit/a5df61561f2f40be88bc0e52bb6e50ce7b6a6052 in the latest development version, which can be seen at https://tweepy.readthedocs.io/en/latest/api.html#tweepy.API.update_profile_banner.
Thanks for the follow-up though.
